### PR TITLE
Fix: Unable to add member to project

### PR DESCRIPTION
**Pull request by mayankagnihotri7**
## Notion card
https://www.notion.so/Can-t-add-a-add-a-team-members-to-a-project-96026c5e630e45f3b33b84c48e4fe3ed

## Summary
- User was not able to add members to a project.

## Preview
[Video](https://www.loom.com/share/ce3be9df351a48dcbf4c98232d7a4a6d)

## Type of change
- Bug fix.

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

- Tested locally.

### Checklist:

- [x] I have manually tested all workflows
- [x] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

File: app/controllers/internal_api/v1/project_members_controller.rb [hunk 1]
```diff
@@ -19,9 +19,12 @@ def add_new_members
 
       return if added_members.blank?
 
-      added_members = added_members.map do |m|
+      added_members = added_members.filter_map do |m|
+        next unless m.key?("hourlyRate")
```
Comment (keshavbiswa) [inline app/controllers/internal_api/v1/project_members_controller.rb:23]: Why are we doing this?

**Comment by supriya3105** [comment]
@apoorv1316 Please test this bug fix.

**Review by apoorv1316 — CHANGES_REQUESTED** [review]
@mayankagnihotri7 I am still not able to add team members to some projects. You can see this behavior in the video. Also, the rates are not being fetched in add team member dropdown for already added users.

**Comment by mayankagnihotri7** [comment]
@apoorv1316 If you have attached any video it is not there in the comment. In my video as far as I can see it is adding the member and fetching the cost when viewing.

**Comment by supriya3105** [comment]
@mayankagnihotri7 Please check the tests that are failing. @apoorv1316 Please retest this functionality and share results

**Comment by supriya3105** [comment]
@mayankagnihotri7 Please resolve conflicts @gowsik-ragunath Can you review this PR today.

**Review by gowsik-ragunath — APPROVED** [review]
LGTM!

**Review by apoorv1316 — CHANGES_REQUESTED** [review]
@mayankagnihotri7 the rates are not being fetched in add team member modal. https://www.loom.com/share/17927c46afaf40608df0216076bc183b 
Also, can you change the heading of the modal from 'Add new client" to "Add team member". cc @supriya3105

**Comment by supriya3105** [comment]
@akhilgkrishnan CI tests are failing. Please check

**Comment by akhilgkrishnan** [comment]
@apoorv1316 and @supriya3105 this PR is ready for testing

**Review by apoorv1316 — APPROVED** [review]
@supriya3105 Verified. Can be merged.

**Pull request merged**